libs/components/i18n/src/lib/modules/i18n/resources.service.spec.ts

```diff
@@ -200,19 +200,25 @@ describe('Resources service', () => {
       resources
         .getString('hi')
         .pipe(take(1))
-        .subscribe(() => {});
```

Comment:
Is the addition of `return` because the function can't be empty? You should be able to remove the function altogether and call `subscribe()` with no parameters.

Reply:
Ya. That was to fix some lint warnings we were getting. I have just removed it and am calling it without any parameter.